dimensiondata/src/main/java/org/jclouds/dimensiondata/cloudcontrol/options/DatacenterIdListFilters.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.dimensiondata.cloudcontrol.options;
+
+import org.jclouds.http.options.BaseHttpRequestOptions;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class DatacenterIdListFilters extends BaseHttpRequestOptions {
+
+   private DatacenterIdListFilters() {
+   }
+
+   private DatacenterIdListFilters datacenterIds(final String... datacenterIds) {
```

Comment:
Make this public, otherwise, if you use the builder to configure the pagination first you won't be able to configure the ids.

Also consider providing the method that accepts a collection of IDs too, as that's convenient since the APIs usually return collections. In my previous comment, I pretended to suggest to "add" the varargs method, not to replace the current one. If you add ti back, consider using `Collection` in the signature instead of `Set`.

Reply:
Ah I misunderstood. Thanks, I will make this change.